packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.js

```diff
@@ -37,11 +38,11 @@ type SearchResultsProps = {
   cohortAddTrace: string => void,
   cohortRemoveTrace: string => void,
   diffCohort: FetchedTrace[],
+  disableComparisions: boolean,
```

Comment:
typo? `disableComparisons` instead of `disableComparisions`

Reply:
Thanks!